scripts/issueEmailGenerator.js

```diff
@@ -99,87 +99,112 @@ if ('issueNumber' in options && typeof options.issueNumber === 'number') {
       }
     }
 
+    const getUrlWithUtmTrackingParams = ({ url, medium = 'newsletter' }) => {
+      const utmParams = {
+        utm_source: 'scriptified.dev',
+        utm_medium: medium,
+      };
+
+      try {
+        const urlWithParams = new URL(url);
+
+        Object.keys(utmParams).forEach(key => {
+          urlWithParams.searchParams.set(key, utmParams[key]);
+        });
+
+        return urlWithParams.toString();
+      } catch (err) {
+        console.error(err);
+        // return url as is if URL is invalid
+        return url;
+      }
+    };
+
     const ogImgURL = getOGImage(currentIssue.title, currentIssue.id, currentIssue.date);
 
+    const issueWebUrl = getUrlWithUtmTrackingParams({ url: `https://scriptified.dev/issues/${currentIssue.id}` });
```

Comment:
Do we need to add this to the issueURL because this is our URL and not an external URL

Reply:
since the URL will be clicked from email, no harm in keeping it I think. Might be helpful in tracking users landing from email since `utm_medium` will be `newsletter` here.

Reply:
Okay